extensions/amp-form/amp-form.md

```diff
@@ -48,7 +48,7 @@ The `amp-form` extension **MUST** be loaded if you're using `<form>` or any inpu
 
 Example:
 ```html
-<form method="post" action="https://example.com/subscribe" target="_blank">
```

Comment:
Unfortunately we still require `target` for all. Feel free to file a bug to remove that requirement from `POST` requests but please keep it in the examples for now.